assets/js/index.js

```diff
@@ -0,0 +1,4 @@
+import * as govukFrontend from 'node_modules/govuk-frontend/dist/govuk/all'
```

Comment:
assets/js/govukFrontendInit.js deleted and replaced by index.js to matche the typescript template's approach for initialising.